aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/options/AWSCognitoAuthConfirmSignInOptions.java

```diff
@@ -90,13 +110,15 @@ public String toString() {
      */
     public static final class CognitoBuilder extends Builder<CognitoBuilder> {
         private Map<String, String> metadata;
+        private List<AuthUserAttribute> userAttributes;
 
         /**
          * Constructor for the builder.
          */
         public CognitoBuilder() {
             super();
             this.metadata = new HashMap<>();
+            this.userAttributes = new ArrayList<AuthUserAttribute>();
```

Comment:
There is no need to explicitly specify the `AuthUserAttribute` type here. Let's remove it for consistency with rest of the codebase.
```suggestion
            this.userAttributes = new ArrayList<>();
```

Reply:
Done.